README.md

```diff
@@ -133,15 +133,15 @@ By default, the gem uses the standard [Data Package Schema](http://specs.frictio
 The gem comes with schemas for the standard Data Package Schema, as well as the [Tabular Data Package Schema](http://specs.frictionlessdata.io/tabular-data-package/), and the [Fiscal Data Package Schema](http://fiscal.dataprotocols.org/spec/). These can be referred to via an identifier, expressed as a symbol.
 
 ```ruby
-package = DataPackage::Package.new(nil, :tabular) # Or :fiscal
+package = DataPackage::Package.new(nil, schema: 'tabular-data-package') # Or 'fiscal-data-package'
```

Comment:
Removing `schema` argument will be in next chunk of changes? Because now it's a part of descriptor - `descriptor.profile`.